src/__test__/containers/SignUpContainer.test.jsx

```diff
@@ -0,0 +1,27 @@
+import configureStore from 'redux-mock-store';
+import React from 'react';
+import { mount } from 'enzyme';
+import { Button } from 'react-materialize';
+import SignUpContainer from '../../containers/Auth/SignUpContainer';
+
+
+const mock = configureStore();
+const store = mock({
+  signUp: {
+    loading: false,
+    success: false,
+  },
+});
+
+describe('test signup container', () => {
+  test('test container mounts', () => {
+    const container = mount(<SignUpContainer store={store} />);
+    expect(container.find(Button).length).toBe(1);
+  });
+  test('on click submit', () => {
```

Comment:
Please have blank lines between each test case to separate them out and improve readability